gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/mapper/PageMapper.java

```diff
@@ -100,6 +101,13 @@ public interface PageMapper {
     @Mapping(target = "uploadDate", source = "createdAt")
     MediaEntity mapMedia(Media media);
 
+    default String map(Object value) {
```

Comment:
Seeing as Object is inherited by many types of objects, this may be employed for lots of different mappings 🙈  Is it necessary for the mapper to work?

Reply:
Yeah, because object is in Open API source.configuration so there was a compilation error without this new mapper:
`Can't map property "Object source.configuration" to "String source.configuration". Consider to declare/implement a mapping method: "String map(Object value)".`

Reply:
For this method, let's try to make it as specific as possible to the object it will be receiving. Can you turn this method into a `@Named` method and call it specifically for mapping the `source.configuration` value?

Reply:
Done